plugin/src/sbt-test/sbt-datatype/growable-schema-java/project/plugins.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("org.scala-sbt" % "datatype-plugin" % "0.1.0-SNAPSHOT")
```

Comment:
Pass in the plugin version via system property -  http://eed3si9n.com/testing-sbt-plugins